<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des seances</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Acceuil</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Rechrcher une seance :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>260</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;  </string>
      </property>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hhhh&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>260</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>üîç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>680</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;  
</string>
      </property>
      <property name="text">
       <string>SupprimerüóëÔ∏è</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>321</width>
        <height>671</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	font: 900 9pt &quot;Segoe UI&quot;;
	font: 900 9pt &quot;Segoe UI&quot;;
	font: 16pt &quot;Segoe UI&quot;;
  
	background-color: rgb(135, 206, 235);
    border: 2px solid #4682B4;
    border-radius: 8px;
    margin-top: 10px;
}
QPushButton {
    background-color: #4682B4;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #5A9BD5;
}</string>
      </property>
      <property name="title">
       <string>‚ûï Ajouter une seance</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>üîëId_seance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>161</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>üìÖDate</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>210</y>
         <width>161</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>üïíHeure de d√©but</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>161</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>‚è≥Dur√©e</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>410</y>
         <width>161</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>460</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>üöóType</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>470</y>
         <width>76</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>Code</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Conduite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parc</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>560</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>560</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>260</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;  </string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Trier par :</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>871</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #00008B;   /* ÿ£ÿ≤ÿ±ŸÇ ÿ∫ÿßŸÖŸÇ */
    color: white;                /* ÿßŸÑŸÜÿµ ÿ£ÿ®Ÿäÿ∂ */
    gridline-color: white;       /* ŸÑŸàŸÜ ÿßŸÑÿÆÿ∑Ÿàÿ∑ */
    border-radius: 8px;          /* ÿ≠ŸàÿßŸÅ ŸÖÿ≥ÿ™ÿØŸäÿ±ÿ© */
}
QHeaderView::section {
    background-color: #34495e;   /* ÿ±ŸÖÿßÿØŸä ÿ∫ÿßŸÖŸÇ ŸÑŸÑŸÄ header */
    color: white;
    padding: 4px;
    border: 1px solid #2c3e50;</string>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Heure d√©but</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dur√©e</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Statut</string>
       </property>
      </column>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>390</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>440</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>490</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>540</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>660</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;  
</string>
      </property>
      <property name="text">
       <string>Exporation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>670</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>‚¨áÔ∏è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>380</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>‚úèÔ∏è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>430</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>‚úèÔ∏è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>480</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>‚úèÔ∏è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>530</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>‚úèÔ∏è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>901</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/WhatsApp Image 2025-09-24 at 15.27.43.jpeg</pixmap>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>590</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>580</y>
        <width>63</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>‚úèÔ∏è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Gestion des s√©ances&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>690</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>1121</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/WhatsApp Image 2025-09-24 at 15.27.43.jpeg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>320</y>
        <width>801</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/WhatsApp Image 2025-09-24 at 15.47.38.jpeg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Statistiques selon les Types</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Autres M√©tiers</string>
     </attribute>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>1121</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/WhatsApp Image 2025-09-24 at 15.27.43.jpeg</pixmap>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>541</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Affectuer une date :</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>591</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(135, 206, 235);</string>
      </property>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>421</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Rappel d'une seances par Mail</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>270</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;  
</string>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>170</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>üìß</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1321</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
